### ISSUE TITLE
Keeping the masks size same as the original Image 

### ISSUE THREAD

**Issue by aborundiya** [facebookresearch/detr#226]
## ❓ How to do something using DETR

Describe what you want to do, including:
1. what inputs you will provide, if any:
    When Run the Detection and Panoptic Segmentation the size of masks are reduced to 1/4th of the images. This makes it bit difficult to do certain augmentations to the original images.  
2. what outputs you are expecting:
   Can you point to resources to get the mask as original image or show how to get that. 

NOTE:

1. Only general answers are provided.
   If you want to ask about "why X did not work", please use the
   [Unexpected behaviors](https://github.com/facebookresearch/detr/issues/new/choose) issue template.

2. About how to implement new models / new dataloader / new training logic, etc., check documentation first.

3. We do not answer general machine learning / computer vision questions that are not specific to DETR, such as how a model works, how to improve your training/make it converge, or what algorithm/methods can be used to achieve X.

**Comment by alcinos** [comment]
Hello,

The target masks are always provided at full resolution, you shouldn't have any issue applying any augmentations you want on them.
It is true that our segmentation head typically predict masks at a 1/4th resolution. If you need something bigger you can simply upsample at will.
This is where we do it in the loss:
https://github.com/facebookresearch/detr/blob/5e66b4cd15b2b182da347103dd16578d28b49d69/models/detr.py#L181-L182
and this is where we do it in the postprocessor:
https://github.com/facebookresearch/detr/blob/5e66b4cd15b2b182da347103dd16578d28b49d69/models/segmentation.py#L224-L236

I believe I have answered your question, and as such I'm closing this. Feel free to reach out if you have further concerns.

**Comment by sodi16** [comment]
@alcinos 
Hii I'm using detr for panoptic segmentation, after I runned 
`panoptic_seg, segments_info = predictor(im)["panoptic_seg"]` , I want to know the label class of each pixel in my segmentation map. It doesn't correspond to the Coco labels class. Thanks in advance
